mowgli_etl/pipeline/wdc/wdc_dimension.py

```diff
@@ -0,0 +1,12 @@
+from typing import NamedTuple, Optional
+
+class Dimension(NamedTuple):
+	depth : Optional[float]
```

Comment:
Still having some formatting issues. This appears to be a single tab. Make sure you're inserting 4 spaces instead of a tab, and running the Black formatter on save.